src/main/drivers/accgyro_spi_mpu6500.c

```diff
@@ -163,8 +167,32 @@ static void mpu6500AccRead(int16_t *accData)
     accData[Z] = (int16_t)((buf[4] << 8) | buf[5]);
 }
 
+#ifdef COLIBRI_RACE
+static void mpu6500SpiInit(void)
```

Comment:
This feels like i should be generic too.

Some drivers are passed a structure that defines the pins used by the target hardware, this is a good approach, because the target specific code is kept out of the driver code.